README.md

```diff
@@ -113,7 +113,7 @@ Login form validation implementation
 **Work in progress: we are working on closing a first release of this library, we expect to register it in npm in the following two three weeks, in the mean time you can download the source code and run the samples following the instructions below.**
```

Comment:
Remove the work in progress, and add installation section like:
https://github.com/reactjs/redux